config/core/deployments/broker.yaml

```diff
@@ -0,0 +1,190 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO(https://github.com/google/knative-gcp/issues/882) Support readiness/liveness probes.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: broker-ingress
+  namespace: cloud-run-events
+  labels:
+    events.cloud.google.com/release: devel
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: cloud-run-events
+      role: broker-ingress
+  template:
+    metadata:
+      labels: *labels
+    spec:
+      # Only needed if running with workload identity.
+      serviceAccountName: broker
+      containers:
+      - name: broker-ingress
+        image: ko://github.com/google/knative-gcp/cmd/broker/ingress
+        env:
```

Comment:
The other two deployments have `SYSTEM_NAMESPACE`, `POD_NAME`, and metrics-related env vars. Why does the ingress container not have them? Seems like we'll need them when we start adding metrics and tracing.

Reply:
The other two currently reports memstats and therefore need metrics domain env var. 

Yes once we add metrics to ingress we will need those env vars as well.  I can add those now or wait until we have metrics added.

Reply:
Either way is fine with me. Feel free to unhold if you'd like to add the env vars later.

Reply:
Sounds good. The binary should fail to start if we forget to add the required env var. So we we will find out easily if we forget to add them